docs/usage.html

```diff
@@ -122,10 +120,23 @@ <h2>Online Mode<a class="headerlink" href="#online-mode" title="Link to this hea
 <section id="command-line">
 <h3>Command line<a class="headerlink" href="#command-line" title="Link to this heading"></a></h3>
 <p>If you want to track the emissions of a computer without having to modify your code, you can use the command line interface:</p>
-<div class="highlight-console notranslate"><div class="highlight"><pre><span></span><span class="go">codecarbon monitor --no-api</span>
-</pre></div>
-</div>
+<p>Create a minimal configuration file (just follow the prompts)
+.. code-block:: console</p>
+<blockquote>
+<div><p>codecarbon config –init</p>
+</div></blockquote>
+<p>Start monitoring the emissions of the computer
+.. code-block:: console</p>
+<blockquote>
+<div><p>codecarbon monitor</p>
+</div></blockquote>
 <p>You have to stop the monitoring manually with <code class="docutils literal notranslate"><span class="pre">Ctrl+C</span></code>.</p>
+<p>In the following example you will see how to use the CLI to monitor all the emissions of you computer and sending everything
+to an API running on localhost:8008 (that you can start with the docke-compose)
```

Comment:
`docker-compose`. 

We could add a `Coming soon` comment or something mentioning that this only works in a local instance of the server for now.